debezium-server/debezium-server-pubsub/src/test/java/io/debezium/server/pubsub/PubSubIT.java

```diff
@@ -89,18 +112,76 @@ public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
 
     void setupDependencies(@Observes ConnectorStartedEvent event) throws IOException {
         Testing.Print.enable();
-
-        try (SubscriptionAdminClient subscriptionAdminClient = SubscriptionAdminClient.create()) {
-            final TopicName topic = TopicName.ofProjectTopicName(ServiceOptions.getDefaultProjectId(), STREAM_NAME);
+        
+        trySetupEmulatorChannel();
```

Comment:
Could you change the name so it will be like `createChannel`.
Then there will be a flow of methods like `createTopic`, `createSubscription` etc.